### PR TITLE
[doc/fpv] Unify the wording for IPs that verified in FPV

### PR DESCRIPTION
**Pull request by cindychip**
This PR unifies the wording in the IP status dashboard.

Signed-off-by: Cindy Chen <chencindy@opentitan.org>

### CODE REVIEW

**Review by tjaychen — APPROVED** [review]

**Review by msfschaffner — APPROVED** [review]

**Review by sriyerg — APPROVED** [review]

File: hw/ip/tlul/data/tlul.prj.hjson [hunk 1]
```diff
@@ -14,14 +14,14 @@
...
-        notes:              "the target for this entry is the autogenerated xbar_main within top_earlgrey."
+        notes:              "The target for this entry is the autogenerated xbar_main within top_earlgrey."
       }
       {
         version:            "1.0",
         life_stage:         "L1",
         design_stage:       "D1",
         verification_stage: "V1",
-        notes:              ""
+        notes:              "Use both FPV and DV to perform block level verification."
```
Comment (sriyerg) [inline hw/ip/tlul/data/tlul.prj.hjson:24]: I would rephrase this to: "Use FPV for the TL building blocks and DV for the generated XBARs."

Comment (cindychip) [reply hw/ip/tlul/data/tlul.prj.hjson:24]: Thanks that sounds good, i will update the notes!
